src/sass/base.scss

```
@@ -1,7 +1,7 @@
 /**
  * The MIT License (MIT)
  *
- * Copyright (c) Camptocamp SA
+// Copyright (c) 2022 Camptocamp SA
```

Shoulfn't be like this
```suggestion
 * Copyright (c) 2022 Camptocamp SA
```

---

Yes :-)